packages/shared/src/components/cards/SharePostCard.tsx

```diff
@@ -57,6 +57,8 @@ export const SharePostCard = forwardRef(function SharePostCard(
   if (improvedSharedPostCard) {
     // eslint-disable-next-line no-param-reassign
     post.image = post.sharedPost.image;
+    // eslint-disable-next-line no-param-reassign
+    post.sharedPost.createdAt = post.createdAt;
```

Comment:
I know you did not do the change but this can have a bunch of edge cases for any component that is using `post` before this component mounts. Better way would be to instantiate new shallow copy and assign the image and createdAt there and then pass it to child components.

Reply:
I think the main issue is it's used a lot down the line in components making it harder to switch.
But can have a quick peek if it's easy to avoid edge-cases